docs/notebooks/atmodel_tests.ipynb

```diff
@@ -0,0 +1,2097 @@
+{
```

Comment:
should this be a part of the codebase? it is a good regression performance test. Could you keep it as a gist elsewhere and keep a link to it in the atmodel.py